TowerDefense/GameObjects/Enemy.cs

```diff
@@ -9,114 +9,88 @@ namespace TowerDefense
 {
     class Enemy : GameObject
     {
-        public string name;
-        public int HP;
-        public int maxHP;
-        public int speed;
+        string name;
+        int HP;
+        int maxHP;
+        int speed;
         List<Node> bestPath;
-        Node temp;
-        public bool spawned = false;
         Point currentDest;
-        public bool dead = false;
-        public bool lose = false;
-        public float scale;
-        public double spawnRate;
+        float scale;
 
         CommandCard commandCard;
 
-        public Enemy(int HP, int speed, Texture2D tex, List<Node> bestPath, String name, float scale, double spawnRate) : base(tex, Point.Zero)
+        public Enemy(int HP, int speed, Texture2D tex, List<Node> bestPath, string name, float scale) : base(tex, Point.Zero)
         {
             this.name = name;
             this.HP = HP;
             this.maxHP = HP;
             this.speed = speed;
             this.scale = scale;
-            this.spawnRate = spawnRate;
             this.bestPath = bestPath;
-
-            this.temp = bestPath[0];
-            this.Position = temp.actualPos;
-            this.bestPath.Remove(temp);
-            this.temp = bestPath[0];
-            currentDest = temp.actualPos;
+            this.Position = currentDest = bestPath[0].actualPos;
 
             commandCard = new CommandCard(name, hp: HP.ToString(), speed: speed.ToString());
         }
+
         public void damage(int damage)
         {
             HP = HP - damage;
             commandCard.HP = HP.ToString();
             ResourceManager.DamagedSound.Play();
-            if ( HP <= 0 )
-                dead = true;
-        }
-        public void spawn()
-        {
-            spawned = true;
         }
-        public void move()
+
+        public bool Update(GameTime gameTime)
         {
-            if (spawned)
+            if (Position.Y > currentDest.Y)
             {
-                if (Position.Y > currentDest.Y)
-                    Position = new Point(Position.X, Position.Y - speed);
-                else if (Position.Y < currentDest.Y)
-                    Position = new Point(Position.X, Position.Y + speed);
-                else if (Position.X > currentDest.X)
-                    Position = new Point(Position.X - speed, Position.Y);
-                else if (Position.X < currentDest.X)
-                    Position = new Point(Position.X + speed, Position.Y);
-
-                else
+                Position = new Point(Position.X, Position.Y - speed);
+            }
+            else if (Position.Y < currentDest.Y)
+            {
+                Position = new Point(Position.X, Position.Y + speed);
+            }
+            else if (Position.X > currentDest.X)
+            {
+                Position = new Point(Position.X - speed, Position.Y);
+            }
+            else if (Position.X < currentDest.X)
+            { 
+                Position = new Point(Position.X + speed, Position.Y);
```

Comment:
That sounds good, except I think your xSpeed is backwards.  I'll change it in a sec.